packages/paste-core/components/anchor/stories/index.stories.tsx

```diff
@@ -1,57 +1,52 @@
 import * as React from 'react';
 import {action} from '@storybook/addon-actions';
-import {withKnobs, select, text} from '@storybook/addon-knobs';
 import {Box} from '@twilio-paste/box';
 import {Stack} from '@twilio-paste/stack';
 import {Heading} from '@twilio-paste/heading';
 import {CustomizationProvider} from '@twilio-paste/customization';
 import {Anchor} from '../src';
 import type {AnchorTargets, AnchorTabIndexes} from '../src/types';
 
-const AnchorTargetOptions = ['_self', '_blank', '_parent', '_top'];
-const AnchorTabIndexOptions = [0, -1];
-
 // eslint-disable-next-line import/no-default-export
 export default {
   title: 'Components/Anchor',
-  decorators: [withKnobs],
   component: Anchor,
 };
 
 export const Default = (): React.ReactNode => {
-  const tabIndexOptions = select('tabIndex', AnchorTabIndexOptions, 0) as AnchorTabIndexes;
-  const targetOptions = select('target', AnchorTargetOptions, '_self') as AnchorTargets;
+  const tabIndexOptions = 0 as AnchorTabIndexes;
```

Comment:
Question: if we're not using knobs, could we just skip passing tabIndexOptions to Anchor? It has tabindex 0 by default.

Reply:
Same question for targetOptions

Reply:
Talked with Jade: since we might be using controls instead of knobs, we will have to do similar stuff with having all the props as variables - so we might as well just leave it as is for now to reduce duplicated work.